pyproject.toml

```diff
@@ -89,7 +89,8 @@ description = "Road-core service is an AI powered assistant that runs on OpenShi
 authors = []
 dependencies = [
     "pdm==2.21.0",
-    "torch==2.5.1",
+    'torch@http://download.pytorch.org/whl/cpu/torch-2.5.1%2Bcpu-cp311-cp311-linux_x86_64.whl ; platform_system != "Darwin"',
```

Comment:
just to keep consistency, better use of double quotes :)

Reply:
I used single quotes here because I needed to use double quotes in the part `; platform_system != "Darwin"',`.  Maybe I could write like `; platform_system != 'Darwin'"',` though... wdyt?

Reply:
Thanks! I have a plan to update the `Containerfile` for further speed-up. I will update the line at that time :-)